### ISSUE TITLE
Multi line selection disappears after Ctrl+Arrow combinations

### ISSUE THREAD

**Issue by aardabalkan** [microsoft/vscode#172500]
Type: <b>Bug</b>

Hi there,
When I select multi-line by pressing Ctrl+Alt+(Up or Down)Arrow keys to start multi-line editing, if I press Ctrl+(Right or Left)Arrow combination to move word by word on my selected lines, the multi-line selection disappears and my multi-cursor turns into single-cursor on the first line that I selected. 
This issue fixes itself sometimes when I restart the VSCode but in the last two days I could not fix this issue after restarting. I hope I was able to explain the issue.

VS Code version: Code 1.74.3 (97dec172d3256f8ca4bfb2143f3f76b503ca0534, 2023-01-09T16:59:02.252Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz (8 x 2419)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|7.80GB (1.66GB free)|
|Process Argv|--crash-reporter-id 3b8862e1-7cf3-45e7-ae45-5be27c071868|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
color-info|bie|0.7.2
vscode-intelephense-client|bme|1.9.4
vscode-tailwindcss|bra|0.9.6
composer-php-vscode|DEV|1.28.12222
vscode-html-css|ecm|1.13.1
prettier-vscode|esb|9.10.4
vscode-pull-request-github|Git|0.56.0
php-debug|xde|1.30.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
pyindex848cf:30577861
nodejswelcome1:30587005
2e4cg342:30602488
f6dab269:30613381

```

</details>

**Comment by gjsjohnmurray** [comment]
Are you able to reproduce the issue after running the command to restart with all added extensions temporarily disabled?

If it doesn't happen in that state I suggest you use the `Start Extension Bisect` command to try and pinpoint which one is causing it.

**Comment by aardabalkan** [comment]
Hi John,

I disabled all of the extensions and tried again, it was same, issue still
there. I have run the Start Extension Bisect command but it was same. After
I uninstalled all of the extensions and it was same. I uninstalled VSCode
and it was same. So, I am using Sublime Text temporarily until the issue
will be resolved.

Additional info: I am using Windows 11 ver.22H2 and Huawei Matebook D16
Core i5-1135G7
I tried by using an external keyboard and it was same... Crazy!

Arda BALKAN


John Murray ***@***.***>, 26 Oca 2023 Per, 12:51 tarihinde
şunu yazdı:
